### ISSUE TITLE
Generated RSA key has invalid exponent?

### ISSUE THREAD

**Issue by mitar** [rakutentech/jwkgen#2]
I generate the key like:

```
$ jwkgen rsa --jwk
```

Example output:

```
{
  "kid": "Urv7E0SAY2vo-fk5cT26i89TmRgWJZAQ4kxRTPq-4-Q",
  "kty": "RSA",
  "n": "q_6BBpjyQHzOFIE13Rv7G5JDL5CqOJpSGVlB9HoiAW4lCdGxnRB5Zbr8FioQwxJYmaEjqQfBXX7ZUYalFdoxKWRcD8ISNc9u5gjOmL_hi0UR0oB7DfiHsQSZKaOM1lXPwF3thM1y7XKoM1ftW7tFHHeJbe5sxzqCjmSqeio-teUnQ_tQSgJ4ck7l8HVbZgqGG4cHGyPpCYyMQ2VvYIr6qL5CElqMQGvwKeRewaSCV9YTiTThLikaueTA4oo6R2F658VfZG8Ds7tuCqqEAnF4J5frhdzpnbQLN5aP9G0yvJBE0Xlqa9G5bD3OJDBLhMcxbddOlM2avAQocIqaQK4ofw",
  "e": "AAAAAAA",
  "d": "Gt5RTbSK86QLlYxx267JVY9Ny5pv3RPOi4M0GHsT4OFSC-ykMIMoU8oYI2ZC5BZznNxdc8RwYqi0TlYYaAeD8pFpCFUq_xj-nBT4IUiodVc-54GGt7IBkJfFTYntMHaLU-PYfQyTTaDBwfOUcuYYb9AEuhWrDJyJVLFJKHh8_3EhV5LtOmpjEeyo27OsXPRj7PgzTwwg4kYZr6woi4nFgQmUotZvY1SZ6jIUNYXmyArik-VyONLLta15YOUf_Jl3VPzTAyqWFaE618RJPJLEfJlD0_O0RANqS1xg_fp69ULzyhaY7qSJlFqTCfSDv6WI9s1WBbYAiSx7eTBEpMKB4Q",
  "p": "0qtuz-qCqOQq3ft13RoYt2FzkIokX7SP_YU5L_c5bVET-wE5Y6tovq5X4cL3cMkV4WvCogWGgymSIKRDFV710EsYPBZ5epzjSseczp2QCjrRF6KI_dy6JcvGEQ8ijWycgpT1Tudvk9QwBt56N-ny0xyAkdk7GX6wpWjIXuNKSNc",
  "q": "0QCplks0dXrX_yVxtsrAIDVybLu4eWs3Xr05_rTTTSjg_xenbdn56QxtXz-8afnuFtcU-b0J5h4FVZ6ViZWeOmD1877XWSyqasA_8Xv8KHopFhIMc5ZgNV4Lf5Y9RQzKkw0Y3Gx98lyWns0ZTamAFjH5pgCI_y9MS1PhSsrIYJk"
}
```

When I try to load it using `gopkg.in/square/go-jose.v2`, I get the following error:

```
crypto/rsa: public exponent too small
```

This is strange because it looks like `AAAAAAA` is pretty large. Maybe too large?

**Comment by boazy** [comment]
It definitely looks incorrect. I'll look into it.
Generally, when generating an RSA key the exponent should be specified by the user as input parameter (perhaps with a reasonable default such as 65537), but I somehow missed that.

**Comment by boazy** [comment]
Solved in 9e84ac4e104ae3474af623d2d74e607cbbcf930c.
Released v1.4.7 to fix this issue.

Please reopen this issue if it's not fixed for you.
